Octokit/Helpers/Enums.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    public enum EnumReaction
```

Comment:
Naming things is hard (™ by @shiftkey :stuck_out_tongue_winking_eye:) but this one kinda grates on me.